### PR TITLE
tikv: add experimental warning for API V2

### PR DESCRIPTION
**Pull request by pingyu**
Signed-off-by: pingyu <yuping@pingcap.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Add experimental warning for TiKV API V2

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/docs-cn/pull/9538

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -419,6 +419,7 @@ RocksDB 多个 CF 之间共享 block cache 的配置选项。当开启时，为
 
 > **警告：**
 >
+> - API V2 是 TiKV 在 v6.1.0 中引入的实验特性，不建议在生产环境中使用。
```
Comment (TomShawn) [inline tikv-configuration-file.md:422]: causal-ts 下面是否也要标注一下呢？

Comment (pingyu) [reply tikv-configuration-file.md:422]: 只有设置 API V2 之后，下边 causal-ts 才是有效的。这种情况下需要也标注吗？

Comment (TomShawn) [reply tikv-configuration-file.md:422]: 如果都是和 API V2 的，建议都说明一下是实验特性吧。

Comment (pingyu) [reply tikv-configuration-file.md:422]: 好的。加上了，PTAL~

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**